src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2126,6 +2126,15 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                                     ->end()
                                     ->prototype('scalar')->end()
                                 ->end()
+                                ->arrayNode('recipients_allowed')
```

Comment:
This is missing the `fixXmlConfig` on the parent node which is required to properly support XML config files where we have multiple tags named in a singular.

And this misses the updated of the XSD file as well.

Reply:
btw, I would name it `allowed_recipients` as adjectives come before the noun in English.